xbmc/windows/GUIMediaWindow.cpp

```diff
@@ -59,6 +59,8 @@
 #include "utils/URIUtils.h"
 #include "utils/Variant.h"
 #include "utils/log.h"
+#include "video/VideoDatabase.h"
+#include "video/VideoDbUrl.h"
 #include "view/GUIViewState.h"
```

Comment:
As you already can see, there is no includes from "video/" as of now and this should stay like this. `CGUIMediaWindow` class is a **generic** base class for video, music, games, pvr, what not... and should not contain any video-only/specific functionality.

Maybe `CGUIWindowVideoBase`is a better fit?